### ISSUE TITLE
Unexpected crosslink shard in sim

### ISSUE THREAD

**Issue by arnetheduck** [status-im/nim-beacon-chain#152]
21e4debda0953148c741ef3a70c391e83a846bc1

Looks like an off-by-one somewhere, epoch numbers don't match:

```
WRN 2019-03-04 14:51:43-06:00 Unexpected crosslink shard                 tid=18654 attestation_data_latest_crosslink="(epoch: 1152921504606846992, shard_block_root: 0000000000000000000000000000000000000000000000000000000000000000)" epoch=18 shard_block_root=0000000000000000000000000000000000000000000000000000000000000000 state_latest_crosslinks_attestation_data_shard="(epoch: 1152921504606846993, shard_block_root: 0000000000000000000000000000000000000000000000000000000000000000)" node=5 
```

[node5.log](https://github.com/status-im/nim-beacon-chain/files/2928227/node5.log)

**Comment by arnetheduck** [comment]
(running tests/simulation/start.sh)

**Comment by arnetheduck** [comment]
fixed by 0.5.0 likely
